### DISCUSSION TITLE
Lose access to $this->get() when change the Feature defaults folder

### DISCUSSION THREAD

**Discussion by wagnerfnds** [pestphp/pest#803] [Q&A] [answered]
Hi guys,

When i run this sample test inside the default Feature folder, it runs ok.

```
it('Should return 200', function () {
    $this->get('/')->assertStatus(401);
});
```

But i've changed my phpunit.xml to point to my controllers folder.

```
<testsuites>
    <testsuite name="Unit">
        <directory suffix="Test.php">./app/Http/Controllers</directory>
    </testsuite>
    <testsuite name="Feature">
        <directory suffix="Test.php">./app/Http/Controllers</directory>
    </testsuite>
</testsuites>
```

When i make this change, and move my tests to controllers folder i loose access to $this->get() function.

Already tried to change the Pest.php file to:
`uses(Tests\TestCase::class)->in('./app/Http/Controllers');`

And also tried to put the uses() on top of my test file.

Things like
`$this->assertTrue(true);`
Works fine...

**Comment by faissaloux** [comment]
use
```php 
uses(Tests\TestCase::class)->in('../app/Http/Controllers');
```

**Reply by wagnerfnds** [reply]
Apreciatte your help!

Still not working, the two main erros that keep change as i made this changes in Pest.php or in my Test file is: 

` Call to undefined method App\Http\Controllers\Login\LoginControllerTest::get(). Did you forget to use the [uses()] function? Read more at: https://pestphp.com/docs/configuring-tests`

and 

`Test case Tests\TestCase can not be used. The folder /app/Http/Controllers/Login/LoginControllerTest.php already uses the test case Tests\TestCase`

**Comment by faissaloux** [comment]
I think you using `Tests\TestCase` twice for the same folder `app/Http/Controllers`, check it out and make sure to `use()` it only once.

**Comment by wagnerfnds** [comment]
Yeah, i think that too, but i'm really using it once.

**Reply by faissaloux** [reply]
This is weird, can you show me your `Pest.php` and `/app/Http/Controllers/Login/LoginControllerTest.php`

**Reply by wagnerfnds** [reply]
For sure. 

Pest.php 
```
<?php

uses(
    Tests\TestCase::class,
)->in('../app/Http/Controllers');

expect()->extend('toBeOne', fn () => $this->toBe(1));

function something(): void
{
    // ..
}

```

LoginControllerTest.php
```
<?php

it('Should return API Unauthenticated', function () {
    $this->get('/login')->assertStatus(401);
});
```

Also my phpunit.xml 
```
<?xml version="1.0" encoding="UTF-8"?>
<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
         xsi:noNamespaceSchemaLocation="./vendor/phpunit/phpunit/phpunit.xsd"
         bootstrap="vendor/autoload.php"
         colors="true"
>
    <testsuites>
        <testsuite name="Unit">
            <directory suffix="Test.php">./app/Http/Controllers</directory>
        </testsuite>
        <testsuite name="Feature">
            <directory suffix="Test.php">./app/Http/Controllers</directory>
        </testsuite>
    </testsuites>
    <source>
        <include>
            <directory suffix=".php">./app</directory>
        </include>
    </source>
    <php>
        <env name="APP_ENV" value="testing"/>
        <env name="BCRYPT_ROUNDS" value="4"/>
        <env name="CACHE_DRIVER" value="array"/>
        <!-- <env name="DB_CONNECTION" value="sqlite"/> -->
        <!-- <env name="DB_DATABASE" value=":memory:"/> -->
        <env name="MAIL_MAILER" value="array"/>
        <env name="QUEUE_CONNECTION" value="sync"/>
        <env name="SESSION_DRIVER" value="array"/>
        <env name="TELESCOPE_ENABLED" value="false"/>
    </php>
</phpunit>

```

**Reply by faissaloux** [reply, marked as answer]
Ow this is because of your `phpunit.xml`, you using the same `directory` for different `testsuites`! Remove one of them since they are the same.

**Reply by wagnerfnds** [reply]
Thats it! Dude you're a hero! Thank you!!
